great-idea-website/css/index.css

```diff
@@ -63,4 +63,116 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Your code starts here! */
\ No newline at end of file
+/* Your code starts here! */
+
+.container {
+	width: 876px;
+	margin: auto;
+}
+
+header nav {
+	display: inline-block;
+	margin-top: 35px;
+}
+
+
+.cover .cover-content {
+	margin-bottom: -9px;
+	display: inline-block;
+}
+
+
+header nav a {
+	margin-right: 35px;
+    margin-left: 27px;
```

Comment:
Make sure you are consistent with the styling. If you are using vscode, you can right click on the file and do "format document"